test/basic-querying.test.js

```diff
@@ -172,7 +177,7 @@ describe('basic-querying', function() {
       User.find({skip: 1, limit: 4, order: 'seq'}, function(err, users) {
         should.exists(users);
         should.not.exists(err);
-        users[0].seq.should.be.eql(1);
+        if (!isCassandraConnector) users[0].seq.should.be.eql(1);
```

Comment:
> To support pagination with Cassandra, it may require changes in the response of find

I'll leave this up to you @raymondfeng and @Setogit to decide.

If we decide to keep the current implementation which does not support paging, then I am proposing to modify the cassandra connector to return an error when the `filter` object contains unsupported `paging` property. That will leave our hands free to implement any solution in the future. What's even more important, the user will be immediately notified that they are doing something unsupported.

In the current implementation, the connectors pretends to understand the request, but returns a response that does not fulfil the operation contract. Users will be confused and we will be wasting our time on support.